Sources/Sham/Extensions/StubbedDataCollection+LaunchEnvironmentEncodable.swift

```diff
@@ -0,0 +1,13 @@
+// Copyright © 2020 SpotHero, Inc. All rights reserved.
+
+import Foundation
+
+extension StubbedDataCollection: LaunchEnvironmentEncodable {
+    /// The key used to save and retrieve the object in the launch environment.
+    public static var launchEnvironmentKey = "mocked-data"
+    // The encoded value of the object in String form, throws an error if unable to encode.
```

Comment:
Can this be broken up?
```suggestion
    public static var launchEnvironmentKey = "mocked-data"
    
    // The encoded value of the object in String form, throws an error if unable to encode.
```